### ISSUE TITLE
Support Promise

### ISSUE THREAD

**Issue by ankorGH** [DevlessTeam/dv-node-sdk#4]
It will be nice to have the api support promises.

**Comment by johnotu** [comment]
@ankorGH Yeah, true. As always PRs are welcome. Could count towards your [Hacktoberfest](https://hacktoberfest.digitalocean.com) contributions if you are participating 😀

**Comment by ankorGH** [comment]
I got a working solution for the project I am working on, I will probably send a PR later today.
